Update developer guide to show review command details

ROHITREDDYBALAM opened the pull request.

ROHITREDDYBALAM committed:
remove email field

ROHITREDDYBALAM committed:
Remove email field

ROHITREDDYBALAM committed:
Merge pull request #1 from AY1920S1-CS2103T-W11-3/master

Update

ROHITREDDYBALAM committed:
Remove phone field

ROHITREDDYBALAM committed:
Remove phone attribute

ROHITREDDYBALAM committed:
Merge branch 'remove-phone'

ROHITREDDYBALAM committed:
Merge pull request #2 from AY1920S1-CS2103T-W11-3/master

Remove phone and email fields (#59)

ROHITREDDYBALAM committed:
Remove phone, email from test  case

ROHITREDDYBALAM committed:
Resolve merge conflicts

ROHITREDDYBALAM committed:
Merge remote-tracking branch 'upstream/master'

ROHITREDDYBALAM committed:
Merge remote-tracking branch 'upstream/master'

ROHITREDDYBALAM committed:
Merge remote-tracking branch 'upstream/master'

ROHITREDDYBALAM committed:
Merge remote-tracking branch 'upstream/master'

ROHITREDDYBALAM committed:
Merge remote-tracking branch 'upstream/master'

ROHITREDDYBALAM committed:
Add details of review command to Developer Guide

jakq commented on docs/DeveloperGuide.adoc:
```diff
@@ -235,12 +235,47 @@ image::CommitActivityDiagram.png[]
+// end::revieweatery[]
 
 === Logging
 
```
rohit can fetch upstream again, because after this there are q alot of differences, somewhere along the line there was some error so need to remerge

jakq commented on docs/DeveloperGuide.adoc:
```diff
@@ -235,12 +235,47 @@ image::CommitActivityDiagram.png[]
+* **Alternative 1 (current choice):** Use a list to store and edit the reviews of an eatery.
+** Pros: Easy for new Computer Science student undergraduates to understand, who are likely to be the new incoming developers of our project.
+** Cons: List is not immutable.
 
```
good to add diagrams as well

ROHITREDDYBALAM committed:
Dg  (#5)

* Rework CloseCommand class (#79)

* Refactor Person to Eatery

v1.0 with 3 EditCommandTest failures.

* Removed 2 test cases from EditCommandTest

* Update BetterModelClassDiagram.puml

* Remove Email and Phone class

* Remove Email Phone multiplicity

* Refactor Person to Eatery edit1

* Add Close Command

* Restore whiteline to AddressBookTest

* Merge remote-tracking branch 'upstream/master' into close-open-eatery

# Conflicts:
#	src/main/java/seedu/address/logic/Logic.java
#	src/main/java/seedu/address/logic/LogicManager.java
#	src/main/java/seedu/address/logic/commands/EditCommand.java
#	src/main/java/seedu/address/logic/parser/AddCommandParser.java
#	src/main/java/seedu/address/logic/parser/ParserUtil.java
#	src/main/java/seedu/address/model/AddressBook.java
#	src/main/java/seedu/address/model/Model.java
#	src/main/java/seedu/address/model/ModelManager.java
#	src/main/java/seedu/address/model/eatery/Eatery.java
#	src/main/java/seedu/address/model/eatery/UniqueEateryList.java
#	src/main/java/seedu/address/model/util/SampleDataUtil.java
#	src/main/java/seedu/address/storage/JsonAdaptedEatery.java
#	src/main/java/seedu/address/storage/JsonSerializableAddressBook.java
#	src/main/java/seedu/address/ui/EateryCard.java
#	src/main/java/seedu/address/ui/MainWindow.java
#	src/main/resources/view/EateryListCard.fxml
#	src/main/resources/view/EateryListPanel.fxml
#	src/main/resources/view/MainWindow.fxml
#	src/test/data/JsonSerializableAddressBookTest/duplicateEateryAddressBook.json
#	src/test/data/JsonSerializableAddressBookTest/invalidEateryAddressBook.json
#	src/test/data/JsonSerializableAddressBookTest/typicalEateriesAddressBook.json
#	src/test/java/seedu/address/logic/LogicManagerTest.java
#	src/test/java/seedu/address/logic/commands/AddCommandIntegrationTest.java
#	src/test/java/seedu/address/logic/commands/AddCommandTest.java
#	src/test/java/seedu/address/logic/commands/ClearCommandTest.java
#	src/test/java/seedu/address/logic/commands/DeleteCommandTest.java
#	src/test/java/seedu/address/logic/commands/EditCommandTest.java
#	src/test/java/seedu/address/logic/commands/FindCommandTest.java
#	src/test/java/seedu/address/logic/commands/ListCommandTest.java
#	src/test/java/seedu/address/logic/parser/AddCommandParserTest.java
#	src/test/java/seedu/address/logic/parser/AddressBookParserTest.java
#	src/test/java/seedu/address/logic/parser/EditCommandParserTest.java
#	src/test/java/seedu/address/logic/parser/ParserUtilTest.java
#	src/test/java/seedu/address/model/AddressBookTest.java
#	src/test/java/seedu/address/model/ModelManagerTest.java
#	src/test/java/seedu/address/model/eatery/EateryTest.java
#	src/test/java/seedu/address/model/eatery/UniqueEateryListTest.java
#	src/test/java/seedu/address/storage/JsonAdaptedEateryTest.java
#	src/test/java/seedu/address/testutil/EateryBuilder.java
#	src/test/java/seedu/address/testutil/EateryUtil.java
#	src/test/java/seedu/address/testutil/EditEateryDescriptorBuilder.java
#	src/test/java/seedu/address/testutil/TypicalEateries.java

* Fix CloseCommand

1. Remove buildClose() from EateryBuilder
2. Add EOF line to EateryListPanel.fxml
3. Fix duplicateEateryAddressBook.json
4. Require nonnull Category for Eatery

* Remove tags from requireAllNonNull in Eatery

* Rework execute method of CloseCommand

- Removed the hasExactEatery method and its calls.

* Merge remote-tracking branch 'upstream/master' into close-open-eatery

# Conflicts:
#	src/main/java/seedu/address/logic/commands/CloseCommand.java
#	src/test/java/seedu/address/logic/commands/AddCommandTest.java
#	src/test/java/seedu/address/logic/commands/CloseCommandTest.java

* Update DeveloperGuide with close feature (#82)

* Update Developer Guide

Updated Appendix A, B, C, D, E, F

* Update Developer Guide

Edited Appendix  C (remove entry from todo).

* Update DeveloperGuide.adoc with Close feature

* Remove UndoSequenceDiagram

* Remove trailing whitespaces from DeveloperGuide

* Update DeveloperGuide.adoc

* Review class (#77)

* remove email field

* Remove email field

* Remove phone field

* Remove phone attribute

* Remove phone, email from test  case

* Resolve merge conflicts

* Add basic version of Review class

* Resolve merge conflicts with upstream

* Fix checkstyle errors

* Fix failing test cases

* Fix failing tests

* Verify test cases checking for similar eateries

* Make reviews Json readable

* Fix checkstyle errors and json adapted review errors

* Make review changes permanent

* Fix checkstyle errors

* Fix checkstyle errors

* Create set reviews method for eatery

* Catch exceptions while parsing review command

* Override equals method in review class

* Improve code quality

* Remove unused imports

* Add `addfeed` command

* Add `deletefeed` command

* Add missing Javadocs

* Add tests for feed-related commands

* Update developer guide with add feature (#87)

* Remove all instances of address

* Add new command: Remark

* Update UserGuide.adoc

* Update UserGuide.adoc

* Update Ui.png

* Update UserGuide.adoc

* Update Ui.png

* Revert "Update Ui.png"

This reverts commit 09529ffa6d15f40b21807f2d1af2b6fd64b99378.

* Revert "Update UserGuide.adoc"

This reverts commit 1011cb16e1d28a6e0b104c2ab2558f326e4deca1.

* Revert "Update Ui.png"

This reverts commit 43aa00b780ab5e7ec30d320bddd7fbb157c98808.

* Revert "Update UserGuide.adoc"

This reverts commit c907562aa13235e5c18bbad56c32118429e03c55.

* Revert "Update UserGuide.adoc"

This reverts commit ea7590c2db054fb1359da7ec92c1364efa3b6da6.

* Revert "Merge branch 'tutorial-adding-command'"

This reverts commit 3f671ae4b369ecc1941d61313756e879f0c52515, reversing
changes made to 5e1abfd36d07a4f508e11d4db75ed375d1c7d091.

* Revert "Remove all instances of address"

This reverts commit 5e1abfd36d07a4f508e11d4db75ed375d1c7d091.

* Update UserGuide.adoc

* Update UserGuide.adoc

* Add activity and sequence diagram for Add command

* Change sequence diagram

* Change activity diagram

* Resize activity diagram

* Update DeveloperGuide.adoc

* Update DeveloperGuide.adoc

* Update DeveloperGuide.adoc

* Update DeveloperGuide.adoc

* Update DeveloperGuide.adoc

* Update DeveloperGuide.adoc

* Update DeveloperGuide.adoc

* Update DeveloperGuide.adoc

* Update developer guide with Mode command details (#84)

* Edit Developer Guide for Mode Command

* Update developer guide

ROHITREDDYBALAM committed:
Merge branch 'master' into dg-ug

ROHITREDDYBALAM committed:
Update dg

ROHITREDDYBALAM committed:
Update Developer guide

ROHITREDDYBALAM committed:
Make changes to dg

jakq reviewed (approved):
To add diagrams for review feature
